src/algorithm/mod.rs

```diff
@@ -21,21 +21,30 @@ impl Algorithm {
         Algorithm::default()
     }
 
+    pub fn get_initial(&self) -> &Node {
+        &self.initial
+    }
+
     /// Set the initial Node to a new state using the provided Goban.
     pub fn update_initial_state(&mut self, initial_state: Goban, last_move: BitBoard, player_captures: u8, opponent_captures:u8) {
         let new_initial_node = Node::new(initial_state, 0, last_move, false, player_captures, opponent_captures);
         self.initial = new_initial_node;
     }
 
-    fn compute_and_set_fscore(&self, node: &mut Node, depth: u32) -> Fscore {
+    // FIXME: SHOULDN'T BE PUBLIC
```

Comment:
normal?

Reply:
Yup, not an issue, just a reminder for good practice. I still need it to be public at this point but we should turn it into a private method when we'll be ready.